README.md

```diff
@@ -34,7 +34,9 @@ Alternatively, you can clone this layer into your own Yocto source tree:
     cd <your work director where other meta* are loaded>
     git clone git://github.com/geontech/meta-redhawk-sdr.git
 
-Then edit your `build/conf/bblayers.conf` to include a reference to `meta-redhawk-sdr` at the end of the list.  See our `meta-redhawk-sdr/conf/bblayers.conf.sample` as an example.
+Then edit your `build/conf/bblayers.conf` to include a reference to `meta-redhawk-sdr` at the end of the list.  See our `meta-redhawk-sdr/conf/bblayers.conf.sample` as an example. 
+
+> Note: The processor architecture is set to `armv7l` by default. If you need a different processor architecture, set the `REDHAWK_PROCESSOR` variable in your `build/conf/bblayers.conf`.
```

Comment:
While it would probably work fine to have it in the BBLayers config, I'm less inclined to suggest people to set global variables in that file since it's the layer configuration for the build.  Instead, I would suggest `build/conf/local.conf` since that's where many of the top-level overrides get set.